ui/app/AppLayouts/Chat/views/ChatContentView.qml

```diff
@@ -375,34 +375,20 @@ ColumnLayout {
             height: chatInput.height
             Layout.preferredHeight: height
 
-            // Not Refactored Yet
-            //            Connections {
-            //                target: chatContentRoot.rootStore.chatsModelInst.messageView
-            //                onLoadingMessagesChanged:
-            //                    if(value){
-            //                        loadingMessagesIndicator.active = true
-            //                    } else {
-            //                        timer.setTimeout(function(){
-            //                            loadingMessagesIndicator.active = false;
-            //                        }, 5000);
-            //                    }
-            //            }
-
-            // Not Refactored Yet
-            //            Loader {
-            //                id: loadingMessagesIndicator
-            //                active: chatContentRoot.rootStore.chatsModelInst.messageView.loadingMessages
-            //                sourceComponent: loadingIndicator
-            //                anchors.right: parent.right
-            //                anchors.bottom: chatInput.top
-            //                anchors.rightMargin: Style.current.padding
-            //                anchors.bottomMargin: Style.current.padding
-            //            }
+            Loader {
+                id: loadingMessagesIndicator
+                active: messageStore.messageModule.loadingHistoryMessagesInProgress
+                sourceComponent: loadingIndicator
```

Comment:
Minor: component may be inlined (instead of creating separate one, line 388)
```suggestion
                sourceComponent: LoadingAnimation { }
```